nextest-runner/src/reporter/displayer/progress.rs

```diff
@@ -517,6 +517,10 @@ fn supports_osc_9_4(stream: &dyn IsTerminal) -> bool {
         debug!("autodetect terminal progress reporting: enabling since TERM_PROGRAM is WezTerm");
         return true;
     }
+    if std::env::var("TERM_PROGRAM").ok() == Some("ghostty".into()) {
```

Comment:
```suggestion
    if std::env::var_os("TERM_PROGRAM").is_some_and(|term| term == "ghostty") {
```

Avoids an allocation (in general it's nice to avoid .into() anyway IMO) and is easier to reason about.

Maybe the other cases can be changed too? And there are 3 ifs that are almost the same, may want to do a single if with a matches!